core/src/main/scala/com/salesforce/op/stages/impl/tuning/Splitter.scala

```diff
@@ -152,7 +171,10 @@ private[op] object SplitterSummary {
 
   def fromMetadata(metadata: Metadata): Try[SplitterSummary] = Try {
     metadata.getString(ClassName) match {
-      case s if s == classOf[DataSplitterSummary].getName => DataSplitterSummary()
+      case s if s == classOf[DataSplitterSummary].getName => DataSplitterSummary(
+        preSplitterDataCount = metadata.getLong(ModelSelectorNames.PreSplitterDataCount),
+        downSamplingFraction = metadata.getDouble(ModelSelectorNames.DownSample)
+      )
```

Comment:
please add the downsample fraction to the datacutter params as well...

Reply:
I added downsample fraction into the datacutter params as part of the multi class classification training limit changes. I'll create the PR for it today.